README.md

```diff
@@ -22,6 +22,13 @@ To build and run, type `make`:
 ```bash
 $ make
 ... lots of configuration nonsense here, downloads, cmake, and then compilation ...
+```
+
+Then to start it up:
+
+```bash
+$ make start
+
```

Comment:
Maybe we could just change to `make start` on line 23 as that will build and start?